api/src/test/java/com/epam/pipeline/test/creator/tool/ToolCreatorUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Rename to docker creator utils as in earlier pull requests for simpler conflicts solving.